normandy/control/urls.py

```diff
@@ -0,0 +1,17 @@
+from django.conf import settings
+from django.conf.urls import url, include
+
+from normandy.control import views as control_views
+from django.contrib.auth.views import login, logout_then_login
+
+urlpatterns = []
+
+if settings.ADMIN_ENABLED:
+  urlpatterns += [
+      url(r'^control/', include([
+          url(r'^$', control_views.IndexView.as_view(), name='index'),
```

Comment:
URL names are global across the whole site. Typically we prefix them with something like `normandy.control.index`. Looking at what we've got thusfar in the project, we haven't really been consistent.

@mythmon Which do you like better as URL names: `normandy.control.index` or just `control.index`? Should we also use [URL namespaces](https://docs.djangoproject.com/es/1.9/topics/http/urls/#url-namespaces-and-included-urlconfs)?